zeze111/andela-bootcamp27
# feature/ratings/component

#### What does this PR do?
Allows users to upvote or downvote a recipe

#### Description of Task to be completed?
 * write the front-end functions for voting 
 * added to the existing html and css code for Recipe details
 * added to the existing style sheet

#### How should this be manually tested?
Have the app working by:
 * run the command `npm runs start:dev` on the command line to start the server
 * once app is running, navigate to the app using `localhost:8000`
 * navigate to a recipe on the home page and click to see details
 * click either the thumbs-up or thumbs-down button to vote on the landing page

 #### Any background context you want to provide?
 Nil

 #### What are the relevant pivotal tracker stories?

 #### Screenshots (if appropriate)
